src/App.tsx

```diff
@@ -1,16 +1,67 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [formInputs, setFormInputs] = useState({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
+  const [allMovies, setAllMovies] = useState([...moviesFromServer]);
```

Comment:
You don't need to copy this, bc you just import this array from another file
```suggestion
  const [movies, setMovies] = useState(moviesFromServer);
```